ui/packages/consul-ui/app/services/data-sink/protocols/http.js

```diff
@@ -2,30 +2,23 @@ import Service, { inject as service } from '@ember/service';
 import { setProperties } from '@ember/object';
 
 export default class HttpService extends Service {
-  @service('settings')
-  settings;
-
-  @service('repository/intention')
-  intention;
-
-  @service('repository/kv')
-  kv;
-
-  @service('repository/session')
-  session;
+  @service('settings') settings;
+  @service('repository/intention') intention;
+  @service('repository/kv') kv;
+  @service('repository/session') session;
 
   prepare(sink, data, instance) {
     return setProperties(instance, data);
   }
 
   persist(sink, instance) {
-    const [, , , model] = sink.split('/');
+    const [, , , , model] = sink.split('/');
     const repo = this[model];
     return repo.persist(instance);
   }
 
   remove(sink, instance) {
-    const [, , , model] = sink.split('/');
+    const [, , , , model] = sink.split('/');
```

Comment:
DataSink will eventually use a `@dataSink` decorator just like `@dataSource` which will mean all of this can go.